src/net/posix-stack.cc

```diff
@@ -960,10 +960,10 @@ std::vector<network_interface> posix_network_stack::network_interfaces() {
                     rtgenmsg gen;
                     ifaddrmsg addr; 
                 }; 
-            } req = { {0}, };
+            } req = {};
 
-            sockaddr_nl kernel = { 0, }; 
-            msghdr rtnl_msg = { 0, };
+            sockaddr_nl kernel = {};
+            msghdr rtnl_msg = {};
```

Comment:
This thing is initialized way below this place, so it make sense to move it and initialize declaration-time instead of zeoifying

Reply:
right. but, again, we don't initialize every field in this struct. for instance, we don't initialize [msg_control](https://elixir.bootlin.com/glibc/latest/source/bits/latest/C/ident/msg_control) . so i think we still need to keep this value-initialization.

Reply:
But if you initialize part of the struct with designated initializers, the missing parts are zeroified by compiler, aren't they?

Reply:
hmm, right. will implement this in the next revision.